lib/.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
IMO, We do not need this file. Better remove it from other SDK repos as well if it's there. @thivi